### ISSUE TITLE
deepomf's figure segmentation dataset

### ISSUE THREAD

**Issue by gwern** [jerryli27/AniSeg#9]
If you're interested in doing further training, deeppomf has an image dataset scraped from yiff.party (includes NSFW images) where PSDs are used to generate pixel-perfect segmentations of figures: https://drive.google.com/file/d/1vK7R_FiD3_pC892sWTy_fOA_HF6E2SKx/view

Artists provide PSDs in layers, where 1 layer is the figure and the rest the background; so pixel-perfect-by-construction segmentation masks can be automatically created for training.

**Comment by simon-on-gh** [comment]
@gwern Awesome! 
Do you have pretrained model on this dataset or scripts to disassemble this PSD files?

**Comment by gwern** [comment]
No, I've never used it. Just deepomf mentioned it as a way to improve on the existing segmentation without extremely laborious hand-annotation or using weak supervision from a classifier/tagger like DeepDanbooru, so I thought I'd mention that this dataset (and approach in general) exists.
